src/ctelnet.cpp

```diff
@@ -1532,16 +1531,16 @@ int cTelnet::decompressBuffer(char*& in_buffer, int& length, char* out_buffer)
         return -1;
     }
     /* TODO: Remove this commented code once it is deemed safe. -MH
-	   else
-	   {
-	    if( zval <= 0 )
-	    {
-	        initStreamDecompressor();
-	        qDebug() << "Listening for new compression sequences";
-	        return -1;
-	    }
-	   }
-	 */
+       else
```

Comment:
It might be okay to take this out now - I dunno how long it has been commented out but it has not shown to be a problem has it?

Reply:
Agree, a separate small PR taking this out would pass fast

Reply:
Doing so in #1772

Reply:
That is now pulled-into `development`.